west.yml

```diff
@@ -109,7 +109,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: d071b4c469cd35bb61f2f1bc08cd68d6d545f2d1
+      revision: pull/708/head
```

Comment:
Please update the revision: 54d59b407921812329d042d7199c38248e27755b

Reply:
Done